src/internal/views/nunjucks/gauging-stations/licences.njk

```diff
@@ -0,0 +1,223 @@
+{% extends "./nunjucks/layout.njk" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "badge.njk" import badge %}
+
+<!--set title-->
+{% set title = [data.stations[data.stationID].riverName] + " at " + [data.stations[data.stationID].label] %}
+
+{% block pageTitle %}
```

Comment:
If the `pageTitle` property is output from the controller, this shouldn't be needed